task/11-katas-2-tasks.js

```diff
@@ -63,7 +115,17 @@ function parseBankAccount(bankAccount) {
  *                                                                                                'characters.'
  */
 function* wrapText(text, columns) {
-    throw new Error('Not implemented');
+    while (text.length) {
+        let i = columns;
+
+        if (text.length > i) {
+            while (text[i] != " ")
+                i--;
+        }
+
+        yield text.substr(0, i);
+        text = text.substr(i + 1);
```

Comment:
method String.prototype.substr is deprecated
Please, use String.prototype.substring